pkg/snmp/traps/forwarder.go

```diff
@@ -62,5 +64,6 @@ func (tf *TrapForwarder) sendTrap(packet *SnmpPacket) {
 		return
 	}
 	log.Tracef("send trap payload: %s", string(data))
+	tf.sender.Count("datadog.snmp_traps.forwarded", 1, "", []string{"snmp_device:" + packet.Addr.IP.String(), "namespace:" + tf.config.Namespace, "snmp_version:" + formatVersion(packet.Content)})
```

Comment:
Makes sense. This would imply adding Namespace as a field of SnmpPacket :
```
type SnmpPacket struct {
	Content   *gosnmp.SnmpPacket
	Addr      *net.UDPAddr
	Namespace string
	Timestamp int64
}
```
so that `snmpPacket.getTags()` can return `snmp_device`, `device_namespace` and `snmp_version` tags.
WDYT?

Reply:
Could work for me honestly. An SnmpPacket in this context is an actual gosnmp packet + information about the source (it's IP address). Namespace is information about the source so I'm fine adding it here.

Maybe one could argue about having a different struct describing the source that would be just the namespace and the IP, both as strings? No strong opiniong here, but no issue on my end to have the namespace in an snmp packet.

Reply:
Totally agree 👍 SnmpPacket now has a `getTags` method that returns associated tags